src/test/FuzzerImpl.cpp

```diff
@@ -455,47 +623,65 @@ class FuzzTransactionFrame : public TransactionFrame
 {
   public:
     FuzzTransactionFrame(Hash const& networkID,
-                         TransactionEnvelope const& envelope)
-        : TransactionFrame(networkID, envelope){};
+                         TransactionEnvelope const& envelope,
+                         AbstractLedgerTxn& ltx)
+        : TransactionFrame(networkID, envelope)
+        , mSignatureChecker(ltx.loadHeader().current().ledgerVersion,
+                            getContentsHash(), mEnvelope.v1().signatures)
+    {
+    }
 
-    void
-    attemptApplication(Application& app, AbstractLedgerTxn& ltx)
+    bool
+    checkValidNoSignature(AbstractLedgerTxn& ltx)
```

Comment:
That's been separated out now too.  In fact, it's out of this PR altogether for now because it was only needed to do the looping in `genFuzz` to ensure valid transactions, and I wasn't finding any clear improvement in metrics from that one.